packages/web/src/components/list/SingleDropdownList.js

```diff
@@ -344,6 +340,10 @@ class SingleDropdownList extends Component {
 		}
 
 		if (!this.hasCustomRenderer && this.state.options.length === 0) {
+			if (this.props.renderNoResults && !this.props.isLoading) {
```

Comment:
`Note For Reviewer`: Added loading condition because on starting when the items are being loaded it would trigger  `renderNoResults` which is not a good UX so adding a condition to bypass this.